index.test-d.ts

```diff
@@ -2,4 +2,6 @@ import {expectType} from 'tsd';
 import hexRgb = require('.');
 
 expectType<hexRgb.RgbaObject>(hexRgb('#cd2222cc'));
-expectType<hexRgb.RgbaTuple>(hexRgb('#cd2222cc', {format: 'array'}));
+expectType<hexRgb.RgbaObject | hexRgb.RgbaTuple | string>(hexRgb('#cd2222cc', {format: 'array'}));
```

Comment:
```suggestion
expectType<hexRgb.RgbaTuple>(hexRgb('#cd2222cc', {format: 'array'}));
```

Same with the others.